utils/results.py

```diff
@@ -18,6 +18,54 @@ def filter_lambda_gc(lambda_gc):
     return (lambda_gc > 0.5) & (lambda_gc < 2)
 
 
+def load_meta_analysis_results(h2_filter: str = 'both', extension: str = 'mt', exponentiate_p: bool = False):
+    """ Wrapper function for get_meta_analysis_results_path that returns the HailTable/MT.
+    Enables creation of a file including both the original meta analysis and the h2 qc pass
+    meta analysis.
+
+    Parameters
+    ----------
+    h2_filter: `str`
+    Can be 'both', 'none', or 'pass'. 'both' loads a merged table; 'none' loads a table with meta
+    analysis results using all phenotype-ancestry pairs; 'pass' loads meta analysis using only those
+    results that pass QC.
+    """
+    def exponentiate_p_tab(tab):
+        tab = tab.annotate_entries(meta_analysis = tab.meta_analysis.map(lambda x: x.annotate(Pvalue=hl.exp(x.Pvalue),
+                                                                                              Pvalue_het=hl.exp(x.Pvalue_het))))
+        return tab
+
+    if extension != 'mt':
+        raise NotImplementedError('Only mt extension implemented for meta analysis.')
```

Comment:
Why parameterize if you don't use? Or do you think you will later?

Reply:
just done here because `get_meta_analysis_results_path` uses it, but you're right I'd rather always use the mt so I'll remove it